spec/locale.html

```diff
@@ -103,18 +104,25 @@ <h1>ApplyOptionsToTag ( _tag_, _options_ )</h1>
               * _region_
               * _variants_
           1. Else,
-            1. Set _languageId_ to _languageId_ with the <emu-not-ref>substring</emu-not-ref> matched by the <code>unicode_region_subtag</code> Unicode locale nonterminal replaced with the string _region_.
-        1. Set _tag_ to _tag_ with the <emu-not-ref>substring</emu-not-ref> matched by the <code>unicode_language_id</code> Unicode locale nonterminal replaced by the string _languageId_.
-        1. Return CanonicalizeUnicodeLocaleId(_tag_).
+            1. Set _languageId_ to _languageId_ with the <emu-not-ref>substring</emu-not-ref> matched by the <code>unicode_region_subtag</code> Unicode locale nonterminal replaced with _region_.
+        1. Set _tag_ to _tag_ with the substring matched by the <code>unicode_language_id</code> Unicode locale nonterminal replaced with _languageId_.
+        1. Return _tag_.
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-apply-unicode-extension-to-tag" aoid=ApplyUnicodeExtensionToTag>
-      <h1>ApplyUnicodeExtensionToTag ( _tag_, _options_, _relevantExtensionKeys_ )</h1>
-
+    <emu-clause id="sec-mergelocaledata" type="abstract operation" oldids="sec-apply-unicode-extension-to-tag">
+      <h1>
+        MergeLocaleData (
```

Comment:
[LocaleData](https://tc39.es/ecma402/#sec-internal-slots) is already a term used in the spec, reusing it here can cause confusion.

Reply:
Good point. Renamed to MakeLocaleRecord.